index.js

```diff
@@ -836,14 +836,52 @@ class ServerlessAppsyncPlugin {
     }, {});
   }
 
+  getRequestMappingTemplate(config, tpl) {
+    const {mappingTemplatesLocation} = config;
+    const {type, field, request=undefined} = tpl;
+    if(request) { // return inline string
+      if(request.toUpperCase().startsWith('VTL:')) {
+        let requestMappingTemplate = request.slice(4).trim();
+        return requestMappingTemplate;
+      } else { // return template at specific path
+        let reqTemplPath = path.join(mappingTemplatesLocation, request);
+        let requestMappingTemplate = fs.readFileSync(reqTemplPath, 'utf8');
+        return requestMappingTemplate
+      }
+    }
+    else { // return template at default path: /mapping-templates/`${type}`/`${field}`/request.vtl
+      let reqTemplPath = path.join(config.mappingTemplatesLocation, `${type}.${field}.request.vtl`);
+      let requestMappingTemplate = fs.readFileSync(reqTemplPath, 'utf8');
+      return requestMappingTemplate;
+    }
+  }
+
+  getResponseMappingTemplate(config, tpl) {
+    const {mappingTemplatesLocation} = config;
+    const {type, field, response=undefined} = tpl;
+    if(response) { // return inline string
+      if(response.toUpperCase().startsWith('VTL:')) {
+        let responseMappingTemplate = response.slice(4).trim();
+        return responseMappingTemplate;
+      } else { // return template at specific path
+        let respTemplPath = path.join(mappingTemplatesLocation, response);
+        let responseMappingTemplate = fs.readFileSync(respTemplPath, 'utf8');
+        return responseMappingTemplate
+      }
+    }
+    else { // return template at default path: /mapping-templates/`${type}`/`${field}`/response.vtl
+      let respTemplPath = path.join(config.mappingTemplatesLocation, `${type}.${field}.response.vtl`);
+      let responseMappingTemplate = fs.readFileSync(respTemplPath, 'utf8');
+      return responseMappingTemplate;
+    }
+  }
```

Comment:
@jzeron Can we make these two methods `DRY`er?

Reply:
Open to suggestions to make lighter...  We are on holiday this week, but can revisit soon to address unit tests & docs...